app/src/main/java/com/kodeco/android/countryinfo/ui/screens/countryinfo/CountryInfoViewModel.kt

```diff
@@ -0,0 +1,67 @@
+package com.kodeco.android.countryinfo.ui.screens.countryinfo
+
+import android.os.Parcelable
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import com.kodeco.android.countryinfo.models.Country
+import com.kodeco.android.countryinfo.repositories.CountryRepository
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+sealed class CountryInfoState : Parcelable {
```

Comment:
This can be moved to a different file since we want to keep viewmodel classes as small as possible.